ddl/column.go

```diff
@@ -31,15 +31,20 @@ import (
 	"github.com/pingcap/tidb/util/types"
 )
 
-func (d *ddl) adjustColumnOffset(columns []*model.ColumnInfo, indices []*model.IndexInfo, offset int, added bool) {
+func (d *ddl) adjustColumnOffset(columns []*model.ColumnInfo, indices []*model.IndexInfo, offset int, lastColOffset int, added bool) {
```

Comment:
The offset of all new added columns is set to lastColOffset  that it is the length of table columns.
LastColOffset is used to differentiate  the old columns and new added columns.